prime-router/src/main/kotlin/Translator.kt

```diff
@@ -133,6 +133,17 @@ class Translator(private val metadata: Metadata, private val settings: SettingsP
         )
         if (routingFilteredReport.isEmpty()) return routingFilteredReport
 
+        // Do conditionFiltering on the routingFilteredReport
+        val conditionFilteredReport = filterByOneFilterType(
+            routingFilteredReport,
+            receiver,
+            organization,
+            ReportStreamFilterType.CONDITION_FILTER,
+            trackingElement,
+            doLogging = true
+        )
+        if (conditionFilteredReport.isEmpty()) return conditionFilteredReport
+
         // Do processingModeFiltering on the routingFilteredReport
         val processingModeFilteredReport = filterByOneFilterType(
             routingFilteredReport,
```

Comment:
This may not matter since I _think_ the list should be empty at this point either way, but this should maybe now be
```suggestion
            conditionFilteredReport,
```
for consistency with each of the other calls.

Reply:
Re-read this, and no it's not empty at this point, so I think this is more of a required change.